x/ibc/02-client/keeper/client_test.go

```diff
@@ -4,103 +4,122 @@ import (
 	"bytes"
 	"fmt"
 
-	"github.com/stretchr/testify/require"
-	tmtypes "github.com/tendermint/tendermint/types"
-
 	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
 	tendermint "github.com/cosmos/cosmos-sdk/x/ibc/07-tendermint"
 	commitment "github.com/cosmos/cosmos-sdk/x/ibc/23-commitment"
+	tmtypes "github.com/tendermint/tendermint/types"
 )
 
 const (
-	testClientType exported.ClientType = iota + 2
+	invalidClientType exported.ClientType = iota + 5
```

Comment:
Maybe add a comment about the reason for this magic number?

Reply:
`iota + 5 = 5` which is just an unregistered client type

Reply:
Can we use `0` for the invalid client type then, perhaps? That way we don't need to change it later.